src/sentry/static/sentry/app/views/settings/components/dataScrubbing/types.tsx

```diff
@@ -128,3 +130,5 @@ export type PiiConfig =
 export type Applications = Record<string, Array<string>>;
 
 export type Errors = Partial<Record<KeysOfUnion<Rule>, string>>;
+
+export type ProjectSlug = Project['id'] | undefined;
```

Comment:
Shouldn't this be `ProjectId` not `Slug`?

Reply:
yes, but the projectId is a slug ... I believe we discussed it during one of the TSC meetings and we decided to use slug in new code

Reply:
but you're picking from `id`